src/ape/managers/query.py

```diff
@@ -38,10 +38,13 @@ def perform_query(self, query: QueryType) -> pd.DataFrame:
 
     @perform_query.register
     def perform_block_query(self, query: BlockQuery) -> pd.DataFrame:
-        blocks_iter = self.chain_manager.blocks.range(query.start_block, query.stop_block)
+        blocks_iter = self.get_blocks(query.start_block, query.stop_block)
         block_dicts_iter = map(partial(get_columns_from_item, query), blocks_iter)
         return pd.DataFrame(columns=query.columns, data=block_dicts_iter)
 
+    def get_blocks(self, start_or_stop: int, stop: Optional[int] = None) -> Iterator[BlockAPI]:
+        return self.chain_manager.blocks.range(start_or_stop, stop)
```

Comment:
I think this should be the other way? Do it the same way you had before, but instead of deleting the method in the `BlockContainer`, call the query method

Reply:
lololol